templates/tease-author.twig

```diff
@@ -0,0 +1,27 @@
+<li id="result-row-{{ post.ID }}" class="media search-result-list-item">
+	<img class="d-flex search-result-item-image" src="{{ post.thumbnail.src('thumbnail') }}"
```

Comment:
Same here, missing js funtctionality, see [here](https://github.com/greenpeace/planet4-master-theme/blob/release-candidate/assets/js/search.js#L58) and [here](https://github.com/greenpeace/planet4-master-theme/blob/release-candidate/assets/js/search.js#L63)